d2l-colors.html

```diff
@@ -44,6 +44,12 @@
 				--d2l-color-white: #fff;
 				--d2l-color-woolonardo: #f2f3f5;
 
+				/* alerts */
```

Comment:
They are probably just for use in alert scenarios. Where should they go? I assumed this was where all of our colors would live...